deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- { include: jslibs.yml }
```

Comment:
I think this needs to go after django is installed and the application is setup since it calls collectstatic this include fails